wp-activate.php

```diff
@@ -18,6 +18,50 @@
 	die();
 }
 
+$valid_error_codes = array( 'already_active', 'blog_taken' );
```

Comment:
@hesyifei any idea why github is showing that I upgraded wp? the file looks exactly the same in the `master` branch so there should be no diff.

Reply:
> @hesyifei any idea why github is showing that I upgraded wp? the file looks exactly the same in the `master` branch so there should be no diff.

I'm not sure either. Have you tried back up your changes in the theme, and then delete and re-clone the repo, and then make changes to the theme?

Reply:
Oh wow, it looks like after waiting for a bit github automatically fixed it. :D